internal/pkg/scaffold/helm/dockerfilehybrid.go

```diff
@@ -60,6 +60,13 @@ RUN  /usr/local/bin/user_setup
 {{- if .HelmCharts }}
 COPY helm-charts/ ${HOME}/helm-charts/{{ end }}
 
+{{ if or .HelmCharts .Watches }}
+# Corrects file permissions to be fully executable and writable
+RUN find ${HOME} -type f -exec chmod -R g+rw {} \; && \
+    find ${HOME} -type d -exec chmod -R g+rwx {} \; && \
+	chmod -R g+rwx /usr/local/bin
+{{ end }}
```

Comment:
Nit: match the other conditional templating:
* dash before the `if` in line 63
* move `{{ end }}` to the end of the previous line